packages/fxa-shared/payments/iap/apple-app-store/purchase-manager.ts

```diff
@@ -65,11 +66,32 @@ export class PurchaseManager {
         throw new Error('No transactions found');
       }
       subscriptionStatus = item.status;
-      // FIXME: improve performance; see https://mozilla-hub.atlassian.net/browse/FXA-4949
       transactionInfo = await decodeTransaction(item.signedTransactionInfo);
       renewalInfo = await decodeRenewalInfo(item.signedRenewalInfo);
+      this.log.debug(
+        'appleIap.querySubscriptionPurchase.getSubscriptionStatuses',
+        {
+          bundleId,
+          originalTransactionId,
+          transactionInfo,
+          renewalInfo,
```

Comment:
These are object literals with primitive values. I learned from jbuck that while BigQuery does allow these objects to be logged as-is, [`mozlog` will stringify them](https://github.com/mozilla/mozlog/blob/119afb7b50b031581ba378e57480a67eecfee25d/lib/formatters/heka.js#L41-L53) before then.